schema/annotation-source.yaml

```diff
@@ -161,30 +151,51 @@ $defs:
     heritable_properties:
+      target_proposition:
+        oneOf:
+        - $ref: "#/$defs/Proposition"
+        - $ref_curie: gks.core:CURIE
+        description: The Proposition about which the Statement is made.
+      evidence_direction:
+         type: string
+         enum: [ "supports", "disputes", inconclusive" ] 
+         description: >-
+           A term indicating the net direction of all evidence assessed against the target proposition. 
       evidence_level:
         $ref_curie: gks.core:Coding
         description: >- 
-          A term indicating the overall strength of support for the Statement based on all evidence assessed.
+          A term indicating the overall strength of support for or against the Statement's proposition, 
+          as provided by all evidence evaluated.
       evidence_score:
         $ref: "#/$defs/DataItem"
         description: >-
-          A quantitative score reflecting the overall strength of support for the Statement based on all 
-          evidence assessed.
-      target_proposition:
-        oneOf:
-        - $ref: "#/$defs/Proposition"
-        - $ref_curie: gks.core:CURIE
-        description: The Proposition about which the Statement is made.
-      conclusion:
+          A quantitative score reflecting the overall strength of support for or against the Statement's 
+          proposition, as provided by all evidence evaluated.
+      confidence_level: 
         $ref_curie: gks.core:Coding
         description: >-
-          The conclusion drawn from the statement proposition, direction, strength, and/or 
-          confidence score.
-      direction:
+          A term indicating the asserting agent's level of confidence that the target 
+          proposition is true or false (the direction this assessment pertains to is captured
+          by the confidence_direction field).
+      confidence_score: 
+        $ref: "#/$defs/DataItem" 
+        description: >-
+          A quantitative score term reflecting the asserting agent's level of confidence that
+          the target proposition is true or false (the direction this assessment pertains to 
+          is captured by the confidence_direction attribute).
+      confidence_direction: 
         type: string
-        enum: [ "supports", "uncertain", "opposes" ]
+        enum: [ "true", "false", "uncertain" ]
         description: >-
-          The direction of this statement with respect to the target proposition.
+          A term indicating whether the assessment reported in the confidence_level and 
+          confidence_score fields reflects confidence that the target proposition is true, 
+          or false. 
+      conclusion:
+        $ref_curie: gks.core:Coding
+        description: >-
+          A term that describes a final conclusion about the subject of the target proposition, 
+          based on what the proposition expresses, and assessments of confidence and/or evidence 
+          level and direction.     
```

Comment:
Updated description of this attribute to fit with updated model and align with VA.